<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutContactDialog</class>
 <widget class="QDialog" name="AjoutContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="PPImgLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Programmes/gestionnaireContact/Images/PPDefault.jpg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ajoutPPbutton">
     <property name="text">
      <string>Ajouter une photo de profil</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="nomLabel">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nomLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="prNomLabel">
       <property name="text">
        <string>prénom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="prNomLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="entrepriseLabel">
       <property name="text">
        <string>Entreprise</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="entrepriseLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="tLPhoneLabel">
       <property name="text">
        <string>Téléphone</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="tLPhoneLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mailLabel">
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="mailLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="validAjoutContactButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="annulAjoutContactButton">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
